aws/resource_aws_security_group_test.go

```diff
@@ -2722,26 +2722,27 @@ func testAccAWSSecurityGroupConfigCidrBlockRuleLimit(egressStartIndex, egressRul
 	c := `
 resource "aws_vpc" "test" {
   cidr_block = "10.1.0.0/16"
+
   tags = {
     Name = "terraform-testacc-security-group-rule-limit"
   }
 }
 
 resource "aws_security_group" "test" {
-  name = "terraform_acceptance_test_rule_limit"
+  name        = "terraform_acceptance_test_rule_limit"
   description = "Used in the terraform acceptance tests"
-  vpc_id = "${aws_vpc.test.id}"
+  vpc_id      = aws_vpc.test.id
 
-	tags = {
+  tags = {
     Name = "tf-acc-test"
   }
 
-	// egress rules to exhaust the limit
-	egress {
-		protocol = "tcp"
-		from_port = "80"
-		to_port = "80"
-		cidr_blocks = [
+  // egress rules to exhaust the limit
```

Comment:
The `#` comment character should be preferred over `//`

Reply:
Fixed!